src/test/java/de/tum/in/www1/artemis/LectureUnitIntegrationTest.java

```diff
@@ -20,6 +20,7 @@
 import de.tum.in.www1.artemis.repository.*;
 import de.tum.in.www1.artemis.util.ModelFactory;
 
+@Deprecated // Moved to Lecture microservice. To be removed
```

Comment:
I don't think we need to deprecate this. We need deprecation to signal to users that they should stop using a certain functionality as it won't get updated anymore and removed in the future.
=> Deprecating tests doesn't do anything as no one "uses" tests.

Reply:
I see your point. I'll remove the @ Deprecated. I will only leave the comment so that it's easy to find later when it's time to remove this file